website/docs/docs/building-a-dbt-project/building-models.md

```diff
@@ -17,6 +17,14 @@ If you're new to dbt, we recommend that you check out our [Getting Started guide
 
 :::
 
+<VersionBlock firstVersion="1.3">
+
+In v1.3, dbt Core is adding support for **Python models**.
+
+dbt's Python capabilities are an extension of its capabilities with SQL models. We recommend that you read this page first, and then read: ["dbt Models (Python)"](python-models)
```

Comment:
Might be worth mentioning also that what says `.sql` below would need to be replaced by `.py` for Python. No need to go into too much detail in that page but just calling it out.

Reply:
I tried a few different permutations of this, but I wasn't happy with any. I'm going to stick with minimal note for now. We have a lot of future work to do, to figure out the right ways to integrate Python capabilities / references into the existing docs that say SQL, SQL, `.sql`